modules/ROOT/pages/roles.adoc

```diff
@@ -197,6 +198,8 @@ Depending on your permissions, you might not see all of these options.
 . Click the *Permissions* drop-down menu, then select the permissions you want to assign to the custom role.
 +
 The list of available permissions is different for each application.
++
+The list of *Permissions* includes all permissions, even if they are already assigned to users. If you add a permission that's already assigned, there is no effect.
 
 . Click the *+* icon to add the permissions to the role.
 
```

Comment:
The steps here are so out of date that it may be best to replace them (possibly starting with the "Click the Permissions tab" up on 184) with a copy of the steps from configure-teams.adoc, of course replacing "team" with "role". If you go this route, note that the "Select the business groups..." step is not applicable to roles, but the "Select which environments..." part is still applicable.